Dockerfile.x86

```diff
@@ -11,7 +11,7 @@ RUN yum install -y cpan
 RUN yum list | grep python
 
 #RUN yum -y install rh-python36
-#RUN yum -y install rh-hython36-pip 
+#RUN yum -y install rh-hython36-pip
```

Comment:
Consider removing all the commented out lines (not just here, but throughout the file) because they make it more difficult to read through this file. If you need to refer to them you can always review your git history.